RCSID $Id: //WIFI_SOC/MP/SDK_4_3_0_0/RT288x_SDK/source/user/openswan-2.4.9/linux/net/Config.in.fs2_0.patch#1 $
--- linux/net/Config.in.preipsec	Mon Jul 13 16:47:40 1998
+++ linux/net/Config.in	Thu Sep 16 11:26:31 1999
@@ -24,4 +24,8 @@
 if [ "$CONFIG_NETLINK" = "y" ]; then
   bool 'Routing messages' CONFIG_RTNETLINK
 fi
+tristate 'IP Security Protocol (Openswan IPSEC)' CONFIG_KLIPS
+if [ "$CONFIG_KLIPS" != "n" ]; then
+  source net/ipsec/Config.in
+fi
 endmenu
